package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fullstory/data-layer-observer",
-  "version": "4.0.4",
+  "version": "5.0.0",
```

Comment:
🤔 Do we need a major version bump? I think all breaking changes (no type suffixes, ignoring `beforeDestination`) are opt-in at the rule definition level.

Reply:
Yeah wasn't sure. I can make it minor release.

Reply:
cc @irv-fs to confirm that we should be good with a minor version bump

Reply:
Talked to @irv-fs He is good with minor.